### DISCUSSION TITLE
(Suggestion) Separating Devs  from active players

### DISCUSSION THREAD

**Discussion by Harudo** [RetroAchievements/RAWeb#794] [Ideas]
So maybe like a filter that you could click or another box that shows people working on achievements for games would make it a lot easier to see/keep track off whats being worked on. Not only that  maybe a little info box saying what else this dev has made/worked on recently in case people get interested in the person doing the set.

**Comment by Jamiras** [comment]
This is similar to #102, which proposes marking sets as in-progress and being able to view in-progress sets.

As for viewing sets a dev has made, you can do that already by going to their profile and clicking the "View all achievement sets <user> has worked on" link. [Here's mine](http://retroachievements.org/gameList.php?d=Jamiras).

**Comment by Tsearo** [comment]
I think separating it on the site feed would be pretty nice. A quick example of how it could look. Also if possible the message for developing achievements could include the name of the game.
![image](https://user-images.githubusercontent.com/4047771/79376249-e45d6700-7f27-11ea-8ab0-b883a76a2b0f.png)

**Comment by FluffyEmmy** [comment]
Yeah, it's a good idea. I was thinking it would look something like searo's mockup (put anyone with Developing/Fixing in Active Devs). Maybe make it tabbed too.

**Comment by HolyShinx** [comment]
Searo made nice example, maybe have it switchable like the side leaderboards from daily to weekly? So like it doesn't take too much space on the main page

**Comment by Jamiras** [comment]
I believe this is handled by the claims feature (#1027) and the activity filter (#682).

It might still be worth separating the "active developers" out of the "active players"widget.
